README.md

```diff
@@ -90,8 +111,16 @@ Define a function called `doSomething`. It should take one argument, called
 `thingToDo`. When called, `doSomething` should invoke the `thingToDo` function. Demonstrate calling `doSomething` while passing in an argument.
 
 Your Answer:
-```js
-// write code here
+```
+function thingToDo() {
+  console.log("I'm doing things!");
+}
+
```

Comment:
This question is a little tricky, but the question was looking for you to pass in an argument into `doSomething`. Here is the solution: 
```

function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```